lib/screens/home_screen.dart

```diff
@@ -0,0 +1,79 @@
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_speed_dial/flutter_speed_dial.dart';
+import 'package:mobile_test/screens/qr_code_screen.dart';
+import 'package:mobile_test/screens/scan_screen.dart';
+import 'package:mobile_test/services/gen_code.dart';
+
+class HomeScreen extends StatefulWidget {
+  const HomeScreen({Key? key}) : super(key: key);
+  @override
+  _HomeScreenState createState() => _HomeScreenState();
+}
+
+class _HomeScreenState extends State<HomeScreen> with TickerProviderStateMixin {
+  // ignore: prefer_typing_uninitialized_variables
+  var _getData;
```

Comment:
**question:** why did you decide to not type this?

Reply:
no particular reason, but yes I should have defined the type since it was known and would be good for readability and for error prevention.